### DISCUSSION TITLE
Error while installing Toltec

### DISCUSSION THREAD

**Discussion by pdharmavaram** [toltec-dev/toltec#549] [Support]
I am getting an error while installing Toltec. Anyone else ran into this

reMarkable: ~/ wget http://toltec-dev.org/bootstrap
Connecting to toltec-dev.org (172.104.246.107:80)
wget: can't open 'bootstrap': File exists
reMarkable: ~/ echo "b41d9bf831b275a12c5ef942599f16a626f84ab79f8c9d0ee8dca2587b8aeb9f  bootstrap" | sha256sum -c && bash bootstrap
bootstrap: OK
INFO:  Bootstrapping Opkg
mkdir: can't create directory '/opt/tmp': No space left on device

**Comment by Eeems** [comment]
Did you ever get around to this again?
